packages/lab/src/collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  type ComponentPropsWithoutRef,
+  forwardRef,
+  type SyntheticEvent,
+  useCallback,
+  useMemo,
+} from "react";
+import { makePrefixer, useControlled } from "@salt-ds/core";
+import { CollapsibleContext } from "./CollapsibleContext";
+import { clsx } from "clsx";
+
+export interface CollapsibleProps
+  extends Omit<ComponentPropsWithoutRef<"div">, "onToggle"> {
+  /**
+   * Whether the accordion is expanded.
+   */
+  expanded?: boolean;
```

Comment:
NIT: it's called a Collapsible and it has expanded props ?

wondered if `open` aligned more with other components ?